tarynmartin/shopify-challenge
# Improve styling

#### What’s this PR do?
- improve styling throughout app
- add media queries to create more responsive app
- improve accessiblity by correctly placing input into label on Search component

#### Where should the reviewer start?
- Changes were made in all CSS files
- Some changes in Results & Nominations.js

#### How should this be manually tested?
- in terminal, run npm start
- in browser, do a search and change the size of the screen
- in dev tools, select the device logo and see how it would look on different devices

#### What are the relevant tickets?
resolves #12 

#### Questions:
- not a question, but would need to do a lot more work to improve display on devices to account for each individual devices dimensions.
